app/src/main/contrib/reducers/estimatesReducer.ts

```diff
@@ -66,7 +66,8 @@ export const estimatesReducer = (state = estimatesInitialState, action: Estimate
                 ...state,
                 hasPopulateSuccess: false,
                 populateErrors: [],
-                populatingSetId: null
+                populatingSetId: null,
+                uploadToken: null
```

Comment:
The token not being re-set was causing the bug you saw with the "has this token been used already" message